common/supplemental/likelySubtags.xml

```diff
@@ -497,7 +501,9 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<likelySubtag from="myz" to="myz_Mand_IR"/>		<!--Classical Mandaic‧?‧?	➡ Classical Mandaic‧Mandaean‧Iran-->
 		<likelySubtag from="mzn" to="mzn_Arab_IR"/>		<!--Mazanderani‧?‧?	➡ Mazanderani‧Arabic‧Iran-->
 		<likelySubtag from="na" to="na_Latn_NR"/>		<!--Nauru‧?‧?	➡ Nauru‧Latin‧Nauru-->
-		<likelySubtag from="nan" to="nan_Hans_CN"/>		<!--Min Nan Chinese‧?‧?	➡ Min Nan Chinese‧Simplified‧China-->
```

Comment:
this is the main change that I think needs to be confirmed.

Reply:
Ah sorry, I can hard-code this back -- let me do that quickly

Reply:
Ah yea, this was going back and forth depending on when you ran the script. I'll keep it as it was -- it but it was subject to population counts and which was the "default" script.

Reply:
I don't know which is correct.

Reply:
Okay, it looks like its ..._Hans_CN in v45, v46, and now fixed in this PR to stay like that.